torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -341,6 +344,9 @@ class TORCH_API ProcessGroupNCCL : public Backend {
     // Clone of opTimeout_ from ProcessGroupNCCL.
     std::chrono::milliseconds opTimeout_;
 
+    // timeout to be reduced after this work finishes.
+    std::chrono::milliseconds opTimeoutReduce_ = std::chrono::milliseconds(0);
```

Comment:
opTimeout totally fails to convey "timeout to be reduced after this work finishes"

Reply:
Do you have any suggestions on a better name? Will `timeoutReduceAfterFinish_` too long?

Reply:
You can make up a new term of art that doesn't have bad connotations, like `ephemeralTimeoutInflight_`, a long name isn't bad either but if you've got some unified concept that shows up a lot of places, having a consistent keyword for it helps a bit.